bin/database.py

```diff
@@ -79,6 +79,55 @@ def update_file_origins(cont_list, cont_name):
     update_file_origins(config.db.sessions.find(query), 'sessions')
     update_file_origins(config.db.acquisitions.find(query), 'acquisitions')
 
+def upgrade_to_3():
+    """
+    scitran/core issue #189 - Data Model v2
+
+    Field `metadata` renamed to `info`
+    Field `file.instrument` renamed to `file.modality`
+    Acquisition fields `instrument` and `measurement` removed 
+    """
+
+    def dm_v2_updates(cont_list, cont_name):
+        for container in cont_list:
+            
+            query = {'_id': container['_id']}
+            update = {'$rename': {'metadata': 'info'}}
+
+            files = container.get('files', None)
```

Comment:
No need to specify the default value here.
The `get` method of a python dict will return `None` if it doesn't find the key anyway.

Please also check the other lines below with default value `None`